src/Mod/Fem/femtaskpanels/task_constraint_bodyheatsource.py

```diff
@@ -97,7 +98,7 @@ def open(self):
     def reject(self):
         self.restore_visibility()
         self.selection_widget.finish_selection()
-        FreeCADGui.ActiveDocument.resetEdit()
+        super().reject()
```

Comment:
I'd probably write all of these as returns, it feels a little strange to discard the return value from `super()` only to immediately recreate it. So, e.g.
```suggestion
        return super().reject()
```

Reply:
@chennes value returned from `super()`